help/C/Help_ch_Transactions.xml

```diff
@@ -43,8 +43,9 @@ Translators:
     </listitem>
 
     <listitem>
-      <para>Importing transactions either from a <acronym>QIF</acronym>/<acronym>OFX</acronym> file or via
-      "<acronym>OFX</acronym> on-line banking".</para>
+      <para>Importing transactions either from an <acronym>OFX</acronym>,
+        <acronym>QFX</acronym>, <acronym>QIF</acronym> or <acronym>CSV</acronym>
+        file or via "<acronym>OFX</acronym> on-line banking".</para>
     </listitem>
```

Comment:
Is it your intention to be exhaustive here ? If you you'd also need to add MT940, MT942 and DTAUS as import formats. Perhaps here is suffices to mention that importing can be done from a number of supported file formats or via online banking. Which those are exactly probably belong in a chapter on importing.